docusaurus/docs/operate/configs/supplier_staking_config.md

```diff
@@ -54,10 +94,60 @@ _`Required`_, _`Non-empty`_
 owner_address: <address>
 ```
 
-The `owner_address` is used as the unique shareholder address for the `Supplier`
+The `owner_address` is the address of the account that owns the `Supplier`s staked
+funds, which will be returned to this account when the `Supplier` unstakes and finishes
+unbonding.
+
+For custodial staking, the `owner_address` is the same as the `operator_address`.
+
+For non-custodial staking, the `owner_address` must be different from the `operator_address`.
+This address receives back the staked `upokt` when the `Supplier` is unstaked.
+
+The `owner_address` can only be changed with a stake message signed by the
+`Supplier`'s owner account.
+
+It is also used as the unique shareholder address for the `Supplier`
 if none of `default_rev_share_percent` or `rev_share_percent` is defined in the
 configuration file.
 
+:::note
+
+The `owner_address` does not identify a `Supplier`; multiple `Supplier`s can have
+the same `owner_address`.
+
+:::
+
+### `operator_address`
+
+_`Optional`_, _`Non-empty`_
```

Comment:
What does "Non-empty" mean and how can not be mutually exclusive with "Optional"?

Reply:
I'm realizing they are are mutually exclusive. Since omitting `operator_address` or having an empty string yields the same effect.